### ISSUE TITLE
Error ERR_PNPM_NO_OFFLINE_TARBALL 

### ISSUE THREAD

**Issue by BlueFox1616** [KaylorBen/nixcord#47]
I get this error when rebuilding on nixos . I saw this was a closed issue but for me it is still a problem       
>  ERR_PNPM_NO_OFFLINE_TARBALL  A package is missing from the store but cannot download it in offline mode. The missing package may be downloaded from https://registry.npmjs.org/js-yaml/-/js-yaml-4.1.0.tgz.
       > Progress: resolved 544, reused 35, downloaded 0, added 0
       For full logs, run 'nix log /nix/store/q65dz53hsl995qgni22jp0g8sns00hqz-vencord-1.10.5.drv'.
error: 1 dependencies of derivation '/nix/store/8rl09vxv6y3wan0xchk3xz50gc9icbmg-discord-0.0.71.drv' failed to build
error: 1 dependencies of derivation '/nix/store/yv29ndryyw08hshkvvvkl6p3czsbj5yl-home-manager-path.drv' failed to build
error: 1 dependencies of derivation '/nix/store/jsk38a6nhi3mlj6p430da184cididnnp-home-manager-generation.drv' failed to build
error: 1 dependencies of derivation '/nix/store/1hx3d0fxmw7w26l0wfnwknnz4nm8mf84-user-environment.drv' failed to build
error: 1 dependencies of derivation '/nix/store/2dbil28hk7ajp6s1ycifk27q74zan54z-etc.drv' failed to build
error: 1 dependencies of derivation '/nix/store/9fisxspgmy86blbjpwc75kdkg967cv1k-nixos-system-nixos-24.11.20241018.4c2fcb0.drv' failed to build

**Comment by BlueFox1616** [comment]
Updating flakes fixed it

**Comment by careb0t** [comment]
@BlueFox1616 what exactly did you do to fix it? I'm getting the same error before and after running `nix update flake`.

**Comment by BlueFox1616** [comment]
Hey @careb0t I upodated flakes and did a rebuild i cant remember doing anything else . If you still have the issue then leave it open . You can ask over at the nixos discord server if anyone can help

**Comment by BlueFox1616** [comment]
There is also the closed issue with the same subject https://github.com/KaylorBen/nixcord/issues/37

**Comment by DontEatOreo** [comment]
This issue will be fixed in https://github.com/KaylorBen/nixcord/pull/48

**Comment by KaylorBen** [comment]
Merged so closing for now.
Set a working vencord version with vesktop.vencord.package in the future to temporarily fix

**Comment by careb0t** [comment]
Is there something I need to do beyond `nix flake update` and `home-manager switch --flake`? I did both of these after the merge and I am still getting the build error with ` ERR_PNPM_NO_OFFLINE_TARBALL  A package is missing from the store but cannot download it in offline mode. The missing package may be downloaded from https://registry.npmjs.org/js-yaml/-/js-yaml-4.1.0.tgz.`. I updated my Nix channels too. Not really sure what else to try.
